blueprints/addon/files/README.md

```diff
@@ -1,26 +1,47 @@
-# <%= addonName %>
 
-This README outlines the details of collaborating on this Ember addon.
+<%= addonName %>
+==============================================================================
 
-## Installation
+[Short description of the addon.]
 
-* `git clone <repository-url>` this repository
+Installation
+------------------------------------------------------------------------------
+
+```
+ember install <%= addonName %>
+```
+
+
+Usage
+------------------------------------------------------------------------------
+
+[Longer description of how to use the addon in apps.]
+
+
+Contributing
+------------------------------------------------------------------------------
+
+### Installation
```

Comment:
Thoughts on having the `Contributing` section largely be a cross-link to `CONTRIBUTING.md`, and moving the guts of this section into that file?

Reply:
thought about it, but given the short length and the content not really fitting with what GitHub seems to expect from that file I decided to keep it in here for now.

Reply:
kk, seems fine either way to me